src/mbgl/map/transform.cpp

```diff
@@ -243,8 +243,8 @@ void Transform::flyTo(const CameraOptions &camera, const AnimationOptions &anima
 
     // When u₀ = u₁, the optimal path doesn’t require both ascent and descent.
     bool isClose = std::abs(u1) < 0.000001;
-    // Perform a more or less instantaneous transition if the path is too short.
-    if (isClose && std::abs(w0 - w1) < 0.000001) {
+    // Perform a more or less instantaneous transition if the path is too short or zoom out factor is inf.
+    if ((isClose && std::abs(w0 - w1) < 0.000001) || isinf(r(0))) {
```

Comment:
> a division by 0 in the following code

So that means either `w1` or `rho2` or `u1` is 0, right? Sorry to keep going here; just want to make sure we aren’t running into something more fundamental.